spec/lib/backup_restore/backuper_spec.rb

```diff
@@ -16,4 +16,25 @@
 
     expect(backuper.send(:get_parameterized_title)).to eq("coding-horror")
   end
+
+  describe '#notify_user' do
+    before { STDOUT.stubs(:write) }
```

Comment:
We can use the helpers [here](https://github.com/discourse/discourse/blob/5a47b351ac97188fe315d663b01e48ac1649abeb/spec/support/helpers.rb#L152-L158) to capture the output instead of stubbing.